docs/user/01-70-switch-to-external-registry.md

```diff
@@ -1,6 +1,6 @@
 # Switch to an external Docker registry at runtime
 
-This tutorial shows how you can [switch to an external Docker registry](../../05-technical-reference/svls-03-switching-registries.md) in a specific Namespace, with Serverless already installed on your cluster. This example relies on the `default` Namespace but you can use any other. You will create a Secret custom resource (CR) with credentials to one of these registries:
+This tutorial shows how you can [switch to an external Docker registry](/docs/user/07-30-switching-registries.md) in a specific Namespace, with Serverless already installed on your cluster. This example relies on the `default` Namespace but you can use any other. You will create a Secret custom resource (CR) with credentials to one of these registries:
```

Comment:
@kwiatekus Since we remove the switching registries doc, should I only remove the link or rephrase this intro a bit?